run-tests.ps1

```diff
@@ -30,7 +30,7 @@ $DotnetInstallDir = "$PSScriptRoot/.dotnet"
 
 Push-Location "$PSScriptRoot\tests\Microsoft.DotNet.BuildTools.Prereqs.Docker.Tests"
 try {
-    Exec "$DotnetInstallDir/dotnet test $OptionalTestArgs --results-directory $PSScriptRoot/artifacts/TestResults"
+    Exec "$DotnetInstallDir/dotnet run --project Microsoft.DotNet.BuildTools.Prereqs.Docker.Tests.csproj --report-xunit-trx --results-directory $PSScriptRoot/artifacts/TestResults $OptionalArgs"
```

Comment:
Using `dotnet run` vs `dotnet test` is the key here per this [guidance](https://xunit.net/docs/getting-started/v3/microsoft-testing-platform)